packages/config/test/store.test.js

```diff
@@ -268,12 +268,12 @@ test('loadConfig', async t => {
   assert.equal(res.app, foo, 'should return app')
 })
 
-test('loadConfig', async t => {
+test('loadConfig with platformatic.yaml', async t => {
```

Comment:
please duplicate this test, not edit it.

Reply:
I think it's a GitHub bug, cause that's what I did already!
Just scroll to line 233 and you'll that test untouched!